README.md

```diff
@@ -649,6 +652,62 @@ And then configure the library to use it:
 config :fun_with_flags, :persistence, adapter: MyApp.MyAlternativeFlagStore
 ```
 
+## Application Start Behaviour
+
+As explained in the [Installation](#installation) section, above in this document, the `:fun_with_flags` application will start automatically when you add the package as a dependency in your Mixfile. The `:fun_with_flags` application starts its own supervision tree which manages all required processes and is provided by the `FunWithFlags.Supervisor` module.
+
+Sometimes, this can cause issues and race conditions if FunWithFlags is configured to rely on Erlang processes that are owned by another application. For example, if you have configured the `Phoenix.PubSub` cache-busting notification adapter, one of FunWithFlag's processes will immediately try to subscribe to its notifications channel using the provided PubSub process identifier. If that process is not available, FunWithFlags will retry a few times and then give up and raise an exception. This will become a problem if you're using FunWithFlags in a large application (e.g. a Phoenix app) and the `:fun_with_flags` application starts much faster than the Phoenix supervision tree.
+
+In these cases, it's better to directly control how FunWithFlags starts its processes.
+
+The first step is to add the `FunWithFlags.Supervisor` module directly to the supervision tree of the host application. For example, in a Phoenix app it would look like this:
+
+```diff
+defmodule MyPhoenixApp.Application do
+  @moduledoc false
+  use Application
+
+  def start(_type, _args) do
+    children = [
+      MyPhoenixApp.Repo,
+      MyPhoenixAppWeb.Telemetry,
+      {Phoenix.PubSub, name: MyPhoenixApp.PubSub},
+      MyPhoenixAppWeb.Endpoint,
++     FunWithFlags.Supervisor,
+    ]
+
+    opts = [strategy: :one_for_one, name: MyPhoenixApp.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+
+  # ...
+```
+
+Then it's necessary to configure the Mix project to not start the `:fun_with_flags` application automatically. This can be accomplished in the Mixfile in a number of ways, for example: (**Note**: These are alternative solutions, you don't need to do both. You must decide which is more appropriate for your setup.)
+
+* Declare the `:fun_with_flags` dependency with either the `runtime: false` or `app: false` options. ([docs](https://hexdocs.pm/mix/1.11.3/Mix.Tasks.Deps.html#module-dependency-definition-options))
```

Comment:
In reading these, I wasn't sure if these were two steps to one solution or two different solutions. I think this might help clarify things a bit:

```suggestion
* **Option A**: Declare the `:fun_with_flags` dependency with either the `runtime: false` or `app: false` options. ([docs](https://hexdocs.pm/mix/1.11.3/Mix.Tasks.Deps.html#module-dependency-definition-options))
```

Reply:
Thanks, that makes sense. I was hoping that the `**Note**: ...` would help with this, but labelling these as options is better.